src/collector/ConfigParser.cpp

```diff
@@ -40,7 +40,8 @@ enum ConfigKey
     Jobs                              = 0x4000,
     Db                                = 0x8000,
     Options                           = 0x10000,
-    ConnectTimeoutMS                  = 0x20000
+    ConnectTimeoutMS                  = 0x20000,
+    NodeBackendStatStaleTimeout       = 0x100000
```

Comment:
Merge issues. I do so in order to minimize diff. All these values are changed in further commits.